### DISCUSSION TITLE
How do i remove flowise branding from bottom of embedded chat?

### DISCUSSION THREAD

**Discussion by codyranney** [FlowiseAI/Flowise#586] [Q&A]
How do i remove flowise branding from bottom of embedded chat?
<img width="424" alt="Screenshot 2023-07-20 at 4 39 15 AM" src="https://github.com/FlowiseAI/Flowise/assets/123781498/8de4c200-709a-4af8-9f5d-c84bd333bed5">

**Comment by chungyau97** [comment]
Hi @codyranney,
Please follow these [steps](https://docs.flowiseai.com/how-to-use#embed).
